DEPRECATED.md

```diff
@@ -19,3 +19,5 @@ a future major Firecracker release, in accordance with our
 - \[[#4428](https://github.com/firecracker-microvm/firecracker/pull/4428)\]
   Booting microVMs using MPTable and command line parameters for VirtIO devices.
   The functionality is substituted with ACPI.
+- \[[#2628](https://github.com/firecracker-microvm/firecracker/pull/2628)\]
+  `seccompiler-bin` `--basic` parameter is deprecated.
```

Comment:
nit:
```suggestion
   The `--basic` flag of `seccompiler-bin`
```

Reply:
Fixed

Reply:
Ah, my main comment was abotu dropping the "is deprecated" tbh. The enumeration starts with "the following things are deprecated", so its kinda duplicated xP

Reply:
Fixed